<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MsgRightWidget</class>
 <widget class="QWidget" name="MsgRightWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>416</width>
    <height>266</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>416</width>
    <height>266</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>416</width>
    <height>266</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MsgRightWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>8</number>
   </property>
   <property name="topMargin">
    <number>8</number>
   </property>
   <property name="rightMargin">
    <number>8</number>
   </property>
   <property name="bottomMargin">
    <number>8</number>
   </property>
   <item>
    <widget class="QWidget" name="MsgFilletWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>250</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>400</width>
       <height>250</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QWidget" name="MsgTopWidget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>400</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>400</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QWidget#MsgTopWidget{background-color: rgb(245, 247, 254);
border-top-left-radius:8px;border-top-right-radius:8px;}</string>
          </property>
          <widget class="QLabel" name="labelIcon">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>9</y>
             <width>22</width>
             <height>22</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>22</width>
             <height>22</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>22</width>
             <height>22</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border-image: url(:/ZcloudMsgPopup/image/msg_right_icon.png);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
          <widget class="QLabel" name="labelTilte">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>10</y>
             <width>200</width>
             <height>20</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>20</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 75 16px &quot;微软雅黑&quot;;
color:#333333;</string>
           </property>
           <property name="text">
            <string>系统消息</string>
           </property>
          </widget>
          <widget class="QPushButton" name="closeButton">
           <property name="geometry">
            <rect>
             <x>358</x>
             <y>5</y>
             <width>32</width>
             <height>28</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>32</width>
             <height>28</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>32</width>
             <height>28</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{border-image: url(:/ZcloudMsgPopup/image/close.png);}
QPushButton:hover,pressed{border-image: url(:/ZcloudMsgPopup/image/close_sel.png);};</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="MsgBottomWidget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>400</width>
            <height>210</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>400</width>
            <height>210</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QWidget#MsgBottomWidget{background-color: rgb(255, 255, 255);border-bottom-left-radius:8px;border-bottom-right-radius:8px;}</string>
          </property>
          <widget class="QLabel" name="labelMsgTilte">
           <property name="geometry">
            <rect>
             <x>68</x>
             <y>14</y>
             <width>264</width>
             <height>24</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>264</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>264</width>
             <height>24</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 75 18px &quot;微软雅黑&quot;;
color:#333333;</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
          <widget class="QLabel" name="labelMsgContent">
           <property name="geometry">
            <rect>
             <x>36</x>
             <y>52</y>
             <width>322</width>
             <height>80</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>322</width>
             <height>80</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>322</width>
             <height>80</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 14px &quot;微软雅黑&quot;;
color:#666666;</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QPushButton" name="moreButton">
           <property name="geometry">
            <rect>
             <x>300</x>
             <y>162</y>
             <width>80</width>
             <height>32</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>32</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>500</width>
             <height>32</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{ background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(2, 164, 253, 1), stop:1 rgba(31, 139, 237, 1));
			border-radius:4px;font: 14px &quot;微软雅黑&quot;;color:rgba(255, 255, 255, 1);}
			QPushButton:hover,pressed{ background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(40, 178, 253, 1), stop:1 rgba(64, 156, 240, 1));
		  border-radius:4px;font: 14px &quot;微软雅黑&quot;;color:rgba(255, 255, 255, 1);}</string>
           </property>
           <property name="text">
            <string>查看更多</string>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ZcloudMsgPopup.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>slot1()</slot>
 </slots>
</ui>
